### PR TITLE
Add Kafka logs in file - Updating sample manifests to utilize helm charts

### PR DESCRIPTION
**Pull request by edaena**
-Add the fabrikate-kafka helm chart
-Enable saving kafka logs in a file

### CODE REVIEW

File: helm-chart/kafka-cluster/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (mtarng) [inline helm-chart/kafka-cluster/Chart.yaml]: This can be more descriptive

**Review by mtarng — COMMENTED** [review]
Oh nice, turning all of the manifests into a helm chart? That'll give the flexibility to turn on/off the external loadbalancer with or without ssl too.

**Review by mtarng — APPROVED** [review]

**Comment by andrebriggs** [comment]
@edaena Looks good. How does one exercise the cluster so that we verify the logs are coming out?

**Comment by edaena** [comment]
@andrebriggs the logs start appear in the file constantly since it's set with log level "INFO". You can connect to the broker pod and cd to the log directory to view the file for now.

**Pull request merged**